tests/drivers/build_all/adc/testcase.yaml

```diff
@@ -65,3 +65,7 @@ tests:
   drivers.adc.test.build:
     platform_allow: qemu_cortex_m3
     tags: adc_test
+  drivers.adc.i2c.build:
```

Comment:
There is already such test case defined for ADS1X1X, so I think these two should be merged into one.